stacktrace-gps.js

```diff
@@ -125,13 +125,18 @@
         return true;
     }
 
+    var _reSourceMapping = /\/\/[#@] ?sourceMappingURL=([^\s'"]+)\s*$/gm;
+
     function _findSourceMappingURL(source) {
-        var m = /\/\/[#@] ?sourceMappingURL=([^\s'"]+)\s*$/m.exec(source);
-        if (m && m[1]) {
-            return m[1];
-        } else {
-            throw new Error('sourceMappingURL not found');
+        var allMatches = source.match(_reSourceMapping);
```

Comment:
Note:
When using a global (g) RegExp with `match` it returns an array with all the matched ocurrences.
But when using it with `exec` it seems to ignore the `global` flag and returns the match groups.